<ui version="4.0" >
 <class>ossimPlanetQtMessageLog</class>
 <widget class="QDialog" name="ossimPlanetQtMessageLog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Message Log</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="theMessageLog" >
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="theClearButton" >
       <property name="text" >
        <string>Clear</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="theCloseButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
